src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.cs

```diff
@@ -1582,7 +1583,6 @@ public async Task PostAsync_ManyDifferentRequestHeaders_SentCorrectly()
                     request.Headers.Add("X-Csrf-Token", "i8XNjC4b8KVok4uw5RftR38Wgp2BFwql");
                     request.Headers.Add("X-Request-ID", "f058ebd6-02f7-4d3f-942e-904344e8cde5");
                     request.Headers.Add("X-Request-ID", "f058ebd6-02f7-4d3f-942e-904344e8cde5");
-
```

Comment:
why is the blank line being deleted?

Reply:
Sorry.. accidental change, was about to push a fix :)